src/components/Pagination/src/Pagination.vue

```diff
@@ -45,21 +45,58 @@ export default {
 </script>
 
 <style lang="scss">
-.pagination {
+@import "@/assets/_variables.scss";
+ .pagination {
   padding-top: 1em;
   text-align: center;
-
+  
+   
   button.btn-prev {
     background: none;
+    padding: 0px;
+    color: $blue;
   }
 
   button.btn-next {
     background: none;
+    padding: 0px;
+    margin-left: 16px;
+    color: $blue;
   }
 
   li.more,
   li.number {
-    background: none;
+    &.active{
+         background:$blue;
+         color:$cochlear;
+         font-weight: bold;
+ }
+  
+    &:first-child{
+      margin-left: 16px;
+    }
+    background: $cochlear;
+    border: 1px solid $blue;
+    margin-left: 10px;
+    border-radius: 50%;
+    width: 24px;
+    height: 24px; 
+    line-height: 24px;
```

Comment:
Change this to 22px which will align the text vertically to the middle. The reason being that the border on the top and bottom has taken up 2px of spaces.